tests/test_install.ts

```diff
@@ -62,12 +62,15 @@ function browserSettings(name: string): BrowserSettings {
     }
 }
 
-const agamaServer = process.env.AGAMA_SERVER || "http://localhost";
-const agamaPassword = process.env.AGAMA_PASSWORD || "linux";
-const agamaBrowser = process.env.AGAMA_BROWSER || "firefox";
-const agamaSlowMo = parseInt(process.env.AGAMA_SLOWMO || '0');
-const agamaHeadless = booleanEnv("AGAMA_HEADLESS", true);
-const agamaInstall = booleanEnv("AGAMA_INSTALL", true);
+const agamaServer: string = process.env.AGAMA_SERVER || "http://localhost";
+const agamaPassword: string = process.env.AGAMA_PASSWORD || "linux";
+const agamaBrowser: string = process.env.AGAMA_BROWSER || "firefox";
+const agamaSlowMo: number = parseInt(process.env.AGAMA_SLOWMO || '0');
+const agamaHeadless: boolean = booleanEnv("AGAMA_HEADLESS", true);
+const agamaInstall: boolean = booleanEnv("AGAMA_INSTALL", true);
+const agamaUser: string = "bernhard";
```

Comment:
an empty line above this one would help here to separate logical blocks.

Reply:
Solved 1c9d18a